src/Hatchinq/Card.elm

```diff
@@ -0,0 +1,323 @@
+module Hatchinq.Card exposing
+    ( Config, Layout(..), Message(..), State, Thumbnail(..), View, configure, expandable, init, layout, update
+    , Title
+    )
+
+{-|
+
+
+# Exposed
+
+@docs Config, Layout, Message, State, Thumbnail, View, configure, expandable, init, layout, update
+
+-}
+
+import Element exposing (Element, centerY, fill, px, shrink)
```

Comment:
A few things are imported from Element - why not include the more common ones, like row, width, column, el, etc? It cuts down on extra code